README.md

```diff
@@ -118,7 +118,7 @@ _If you’re unsure of how to update the Open Liberty navigation or unfamiliar w
 
 Since the Open Liberty [draft](https://draft-openlibertyio.mybluemix.net/), [staging](https://staging-openlibertyio.mybluemix.net/) and [production](https://www.openliberty.io/) sites each maintain unique navigation files, you must update them individually, otherwise, unwanted changes and merge conflicts can occur.
 
-The `draft` and `staging`branches each have a dedicated branch for editing the navigation. For the draft branch, this branch is called `draft-nav` and for the staging branch it’s called `staging-nav`. When you’re working in these branches, edit only the `nav.adoc` file and open pull requests only to the branch for which they are specified: `draft` for `draft-nav` and `staging` for `staging-nav`. The production navigation is automatically updated from `vNext` with each release. _You should never update the nav.adoc file in the `vNext` branch_. This file is updated each time the staging branch is pulled to vNext to prep for a release.
+The `draft` and `staging` branches each have a dedicated branch for editing the navigation. For the draft branch, this branch is called `draft-nav` and for the staging branch it’s called `staging-nav`. When you’re working in these branches, edit only the `nav.adoc` file and open pull requests only to the branch for which they are specified: `draft` for `draft-nav` and `staging` for `staging-nav`. The production navigation is automatically updated from `vNext` with each release. _You should never directly update the nav.adoc file in the `vNext` branch_. This file (along with everything else in `vNext` branch), is only ever updated through PRs from the `staging` branch.
```

Comment:
LGTM- needs a slight edit for ID style but I can make that once it's merged:

This file, along with everything else in `vNext` branch, is only updated through PRs from the `staging` branch.